src/todos/todos.database.service.ts

```diff
@@ -1,10 +1,56 @@
-import { Injectable, NotFoundException } from '@nestjs/common'
-import { initialTodos } from './mocks'
+import {
+  Injectable,
+  InternalServerErrorException,
+  NotFoundException,
+  OnModuleInit,
+  OnModuleDestroy,
+} from '@nestjs/common'
+import * as path from 'path'
+import * as fs from 'fs/promises'
```

Comment:
TBD if it's a correct way to import these packages

Reply:
Nowadays it's preferable to prefix Node's modules with `node:` prefix: `import * as fs from 'node:fs/promises'`
Also, it's always better to import only that parts that you need: `import { readFile } from 'node:fs/promises'` — unless you need most of the exported entities.

Reply:
Thanks! I updated these imports as you advised